### DISCUSSION TITLE
0.10.1

### DISCUSSION THREAD

**Discussion by rcap107** [skrub-data/skrub#2264] [Announcements]
# 🌟 Skrub 0.10.1 has been released 🌟 

## Main features
- When a cross-validation splitter has been passed to `.skb.mark_as_X()`, it it now possible to select a specific split from it when calling `.skb.train_test_split()` by passing `split_index`; for example data_op.skb.train_test_split(split_index=3) to get the third split. #2213 @jeromedockes.
- Added support in `tabular_pipeline()` for estimators instantiated from either [tabicl.TabICLClassifier](https://tabicl.readthedocs.io/en/stable/api.html#tabicl.TabICLClassifier) or [tabicl.TabICLRegressor](https://tabicl.readthedocs.io/en/stable/api.html#tabicl.TabICLRegressor) with recommended default parameters of `TableVectorizer` as the first step, and the estimator as the second step. #2222 by @ashwinvis.

## Changes
- `TextEncoder`’s verbose parameter is now an int instead of a bool, where verbose=0 silences the progress bar and verbose>=1 shows it. The default is now 0. #2249 by @Jayant-kernel. 
- `patch_display()` now uses a minimal, faster TableReport without plots or column associations by default. #2103 by @ashiorkornortey. 

## Bugfixes
- `DropSimilar` now works with Polars dataframes when PyArrow is not installed. #2216 by @ShreyanshGoyal.
- The parallel coordinate plot created by `ParamSearch.show_results()` could have incorrect tick labels in some cases. This has been fixed in #2215 by @jeromedockes.
- `GapEncoder` with init="k-means" raised an error when the input column contained missing values. This has been fixed in #2238 by @Hrafz.
- `DatetimeEncoder` no longer raises `UnboundLocalError` when resolution=None is combined with periodic_encoding="circular" or "spline", and no longer fits an unused weekday periodic encoder when resolution is finer than "hour". #2240 by @Hrafz.
- When cols was not provided, `AggJoiner` and `MultiAggJoiner` selected the columns to aggregate through a Python set, so the order of the aggregated output columns varied between runs. They now keep the order in which the columns appear in the auxiliary table. This has been fixed in #2250 by @dylanpulver.
- A cloudpickle import error that could happen after updating some required dependencies was fixed in #2261 by @jeromedockes and @rcap107.

**Full changelog:** https://github.com/skrub-data/skrub/compare/0.10.0...0.10.1

<hr /><em>This discussion was created from the release <a href='https://github.com/skrub-data/skrub/releases/tag/0.10.1'>0.10.1</a>.</em>
